tests/test_endpoints.py

```diff
@@ -116,3 +117,32 @@ def test_nhs_login_p9(nhsd_apim_proxy_url, nhsd_apim_auth_headers):
         params=params
     )
     assert resp.status_code == 200
+
+
+@pytest.mark.smoketest
+@pytest.mark.auth
+@pytest.mark.integration
+@pytest.mark.user_restricted_separate_nhs_login
+@pytest.mark.nhsd_apim_authorization(
+    {
+        "access": "patient",
+        "level": "P9",
+        "login_form": {"username": "P9"}
+    }
+)
+def test_prism_returns_external_file(nhsd_apim_proxy_url, nhsd_apim_auth_headers):
+    headers = {
+        "accept": "application/fhir+json",
+        "X-Correlation-ID": "11C46F5F-CDEF-4865-94B2-0EE0EDCC26DA",
+        "X-Request-ID": "60E0B220-8136-4CA5-AE46-1D97EF59D068",
+        "Interaction-ID": "urn:nhs:names:services:gpconnect:fhir:operation:gpc.getstructuredrecord-1"
+    }
+    headers.update(nhsd_apim_auth_headers)
+
+    resp = requests.get(
+        f"{nhsd_apim_proxy_url}/Appointment/1",
+        headers=headers
+    )
+
+    expected_response = load_example("../specification/examples/single_appointment.yaml")
```

Comment:
the `path` that you're passing is already relative to the `specification` directory. So this should be `examples/single_appointment.yaml`.

Reply:
But its from the tests directory not the specification one ?